components/homme/src/theta-l_kokkos/cxx/ForcingFunctor.hpp

```diff
@@ -472,14 +471,14 @@ class ForcingFunctor
         } else {
           // Compute hydrostatic p from dp. Store in exner, then add to pnh
           auto p_i = Homme::subview(m_pi_i,kv.team_idx,igp,jgp);
-          m_elem_ops.compute_hydrostatic_p(kv,dp,p_i,exner);
+          m_elem_ops.compute_hydrostatic_p(kv,dp_adj,p_i,exner);
```

Comment:
Here is the change to make F90 BFB. We were not using the correct dp calculation which matches F90.